src/library/Select/Select.js

```diff
@@ -7,6 +7,7 @@ import scrollIntoViewIfNeeded from 'scroll-into-view-if-needed';
 import { createStyledComponent, pxToEm } from '../styles';
 import { createThemedComponent, mapComponentThemes } from '../themes';
 import { composeEventHandlers, generateId, isRenderProp } from '../utils';
+import ModifiersContext from '../Dialog/ModifiersContext';
```

Comment:
This work is really neat. Thanks for the update! I know we had discussed importing from a more a general directory so that if we need other kinds of context later, we could just add it to the one we're creating here. I'm totally ok with leaving it for now until we have a need to add more, but just want to make sure my understanding is correct. Or is it possible/preferable to wrap with multiple different context consumers?